interfaces/cython/cantera/reactor.pyx

```diff
@@ -804,7 +854,18 @@ cdef class Valve(FlowDevice):
         >>> V = Valve(res1, reactor1)
         >>> V.set_valve_coeff(1e-4)  # Set the value of K to a constant
         >>> V.set_valve_coeff(lambda dP: (1e-5 * dP)**2)  # Set the value of K to a function
+
+        .. deprecated:: 2.5
+
+             To be deprecated with version 2.5, and removed thereafter.
+             Replaced by property to be consistent with other coefficient definition
+             (e.g. Walls). Doc string for set_valve_coeff is further not consistent
+             with C++ implementation (Func1 value is not handled as a coefficient,
+             making its usage non-intuitive).
```

Comment:
In general, I would suggest that the deprecation notation does not need to go into the rationale for the deprecation, but just provide the recommended alternative.

Reply:
Ok.